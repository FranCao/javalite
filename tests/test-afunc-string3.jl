int main()
{
    string x;
    string l;
    int len_x;
    
    x = "ThIS is A vERy loNG SENTence THat I AM writing";
    print(x);
    l = lower(x);
    print(l);

    len_x = len(x);
    print(len_x);
}