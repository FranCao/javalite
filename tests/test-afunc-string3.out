ThIS is A vERy loNG SENTence THat I AM writing
this is a very long sentence that i am writing
46